docs-crawler/docs/docs/spiders/tf_spider.py

```diff
@@ -0,0 +1,50 @@
+import re
```

Comment:
Most of the files here are generated by Scrapy, however these "spider" files are the main components doing web scraping of the docs and formatting them into json blobs or "Items"